numba_cuda/numba/cuda/tests/cudapy/test_inspect.py

```diff
@@ -27,6 +27,16 @@ def test_monotyped(self):
 
         @cuda.jit(sig)
         def foo(x, y):
+            """
+            // LLVM: foo
+            // LLVM-LABEL: entry:
+            // LLVM-NEXT:         br label %"[[VAL_0:.*]]"
+            // LLVM-NEXT:       [[VAL_0]]:
```

Comment:
Is `[[VAL_0]]` a placeholder for jump label? It just needs matching labels between this line and previous line right?

Reply:
That is right - with FileCheck we can create _captured variables_ the first time a pattern is encountered, and re-use those variables later on to ensure that values we see in one part of the text also appear in another.